OneLogin.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,4 +1,5 @@
 {
+  "originHash" : "1fa7c85448908223f4eead79fe01e6cc23c9840495325eaa8c971fb885706179",
```

Comment:
Sorry to be a pain but can you explain what this is and why we need it? Xcode keeps trying to remove it locally for me so I assume it's something we've added manually?

Reply:
I have no idea, honestly. I saw it when I was trying to fix the Package.resolved issues but just assumed it had to be there. I know it isn't in the one that's on Develop though, so I'm happy for it to come out if we can't think of a good reason why it should be there (beyond Xcode saying it needs to be for me and not for you)

Reply:
If you do decide to remove it (I'm also happy to if you want), just double check the `version` at the bottom of the file is still 2, if it changes to 3 bad things happen it seems

Reply:
Got ya, might be something to read up on, I might do a bit of testing on it quickly just to make sure it's fine without it and nothing breaks 👍

Reply:
I can't really find much on it beyond it being some sort of versioning/integrity checking device. Apparently it's more commonly found within specific dependencies (e.g. as a property in a specific packages `state` section) but not how it is here. Maybe it came about from switching branches or something?

Reply:
Seems to be fine without it 🙂